doc/src/Python_scatter.rst

```diff
@@ -23,19 +23,19 @@ passed by all calling processors, to individual atoms, which may be
 owned by different processors.
 
 Note that the data returned by the gather methods,
-e.g. gather_atoms("x"), is different from the data structure returned
-by extract_atom("x") in four ways.  (1) Gather_atoms() returns a
-vector which you index as x[i]; extract_atom() returns an array
-which you index as x[i][j].  (2) Gather_atoms() orders the atoms
-by atom ID while extract_atom() does not.  (3) Gather_atoms() returns
-a list of all atoms in the simulation; extract_atoms() returns just
-the atoms local to each processor.  (4) Finally, the gather_atoms()
+e.g. :code:`gather_atoms("x")`, is different from the data structure returned
```

Comment:
```suggestion
e.g. :py:meth:`gather_atoms("x") <lammps.lammps.gather_atoms()>`, is different from the data structure returned
```
Using ":py:meth:" as shown here should insert a link to the documentation of the method.